### DISCUSSION TITLE
Welcome to cryoDRGN 🐉 ❄️ Discussions!

### DISCUSSION THREAD

**Discussion by zhonge** [zhonge/cryodrgn#78] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and learn from each other as users of cryoDRGN ❄️🐉. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kimdn** [comment]
Hello, 

My name is Doonam and I automate cryo-EM data processing.

My curiosity is when cryodrgn2 code will be released (alpha version is fine)

https://openaccess.thecvf.com/content/ICCV2021/html/Zhong_CryoDRGN2_Ab_Initio_Neural_Reconstruction_of_3D_Protein_Structures_From_ICCV_2021_paper.html

**Comment by zhonge** [comment]
Hi everyone! I'm Ellen Zhong (@zhonge). I'm the main developer and maintainer of cryoDRGN. I'm hoping that this will be a useful place to organize the questions I've received about training and interpreting cryoDRGN models, share cool results and tips, and announce updates and new developments to the software!

**Comment by Guillawme** [comment]
Hello! My name is Guillaume (@Guillawme online). I am a structural biologist using cryoEM to study the structure of nucleosomes in complex with various factors.

Studying molecular machines like ATP-dependent chromatin remodelers inevitably leads you into "structural dynamics" territory (i.e. there is no such thing as a single, static structure), so I am very interested in cryoDRGN and related methods to analyze heterogeneous ensembles. The first time I needed cryoDRGN, its method paper had just been published, and it worked great on the data I was working on, so I will be forever excited about it!

I don't code much (although I would like to develop my skills), but I now have a lot of mileage with installing and troubleshooting research software packages (cryoDRGN is one of the easy ones to deal with 😇 ), so I try to report bugs when I encounter them (sometimes with a solution or a workaround, if it doesn't require actual coding or is simple enough to fix).
